README.md

```diff
@@ -286,9 +287,10 @@ Submit a task to AO and get the result. The combination of submitTask and getRes
 
 ### getComputationPrice
 
-Get the computing price of each node for each task. Now only supports AO’s test token AOCRED, minimum unit to use AOCRED(1 means 0.001 AOCRED).
+Get the computing price of each node for each task. Now only supports wAR(the Wrapped AR in AO), minimum unit to use wAR(1 means 0.000000000001 wAR).
 
-* **Parameters:** NULL.
+* **Parameters:** 
+  * `symbol:number(optional)` The price symbol (default: wAR).
```

Comment:
symbol:number should be symbol:string

Reply:
Fixed. See [363fbcb](https://github.com/pado-labs/pado-ao-sdk/pull/57/commits/363fbcbd5d85fb9c24f4e910e339a9cb18cdd95c).